tendermint/tests/support/rpc/genesis.json

```diff
@@ -20,6 +20,7 @@
           ]
         }
       },
+      "validators": [],
```

Comment:
Do you mind adding in a validator here? You copy&paste the from genesis example in https://docs.tendermint.com/master/rpc/#/Info/genesis

Reply:
Done